internal/commands/pull/command.go

```diff
@@ -108,6 +131,12 @@ func (cmd *Command) Inputs() cli.InputResolver {
 
 // Handler is the command handler
 func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.Clients) error {
+	if cmd.inputs.IncludeNodeModules && cmd.inputs.IncludePackageJSON {
+		return fmt.Errorf("please exclusively use the --include-package-json flag to upload the " +
```

Comment:
we can probably trim down this error message a bit too, b/c once we're returning it from the `Resolve` function a lot of these same words will appear in the flags' usage strings.

```suggestion
		return errors.New(`must not use both "--include-package-json" and "--include-node-modules" at the same time`)
```